[Bug]

- npx ccusage-widget 는 문제없이 작동함
- 다음과 같이 실행했을때 문제가 발생해
  ```
  npm install -g ccusage-widget
  ccusage-widget
  ```


에러 내용
```
❯ ccusage-widget

node:internal/modules/cjs/loader:1372
  throw err;
  ^

Error: Cannot find module 'electron'
Require stack:
- /Users/jaesoon-jeong/.local/share/mise/installs/node/24.3.0/lib/node_modules/ccusage-widget/bin/ccusage-widget.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1369:15)
    at defaultResolveImpl (node:internal/modules/cjs/loader:1025:19)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1030:22)
    at Module._load (node:internal/modules/cjs/loader:1179:37)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)
    at Module.require (node:internal/modules/cjs/loader:1449:12)
    at require (node:internal/modules/helpers:135:16)
    at Object.<anonymous> (/Users/jaesoon-jeong/.local/share/mise/installs/node/24.3.0/lib/node_modules/ccusage-widget/bin/ccusage-widget.js:7:22)
    at Module._compile (node:internal/modules/cjs/loader:1692:14) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/jaesoon-jeong/.local/share/mise/installs/node/24.3.0/lib/node_modules/ccusage-widget/bin/ccusage-widget.js'
  ]
}

Node.js v24.3.0
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ccusage-widget",
-  "version": "1.2.1",
+  "version": "1.2.2",
   "description": "A beautiful macOS desktop widget that displays your Claude Code usage statistics in real-time",
   "main": "dist/main.js",
   "bin": {

